### PR TITLE
APIM 3.20.0 changelog

### PR DESCRIPTION
**Pull request by AndrewMcFarlandCampbellGravitee**
**Issue**

https://gravitee.atlassian.net/browse/DOC-299

**Description**

Initial draft of changelog for 3.20

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/doc-299/index.html)

### CODE REVIEW

File: pages/apim/3.x/changelog/changelog-3.20.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+** Subscribe to an API with a webhook entrypoint
+* Basic httpproxy for v4 - Sync API
+* Support conditions on messages
+
+=== Gravitee Kubernetes Operator (Beta)
+
+* Re-apply API CRD on ManagementContext update
+* Store credentials in K8 secrets
+* Export a complex API and import it in a new environment with GKO
+* Export to CRD - different options
```
Comment (mouligno) [inline pages/apim/3.x/changelog/changelog-3.20.adoc:51]:
```suggestion
=== API Management - Console

* Brand new menu to unify the experience between all Gravitee products
* Introducing v1 APIs reaching end of support and help users migrate to v2
* APIs list page navigation enhancements
* APIs are read-only if managed by our GKO (Gravitee Kubernetes Operator)
* Allow to delete a media using the Management API
* [BETA v4 M-API] Manage v4 APIs subscriptions as an API Publisher & an API Consumer

=== Event-Native API Management (Beta)

* Endpoint - Kafka Connector Advanced (EE)
** Ability to connect to a secured Kafka cluster
** Introducing RESUME & LIMIT capabilities for QoS (Quality of Service)
* Endpoint - MQTT5 Connector
** Consume messages from a MQTT5 event-broker
** Push messages to a MQTT5 event-broker
* Endpoint - MQTT5 Connector Advanced (EE)
** Ability to connect to a secured MQTT5 event-broker
** Introducing QoS (Quality of Service) capabilities
* Entrypoint - Webhook connector
** Create an API with a webhook entrypoint
** Subscribe to an API with a webhook entrypoint
** Receive messages on my callback url 
* Introducing foundation support for sync API on v4 - httpproxy entrypoint & endpoint
* Message filtering policy
* Support conditions on messages

=== Gravitee Kubernetes Operator (Beta)

* CRD Lifecycle - Apply changes related CRDs on update
* Store credentials in K8 secrets
* Export a complex API and import it in a new/same environment 
* Manage resources as CRD and reuse them in several APIs
```

File: pages/apim/3.x/changelog/changelog-3.20.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+== APIM - 3.20 (2022-12-30)
+
+=== API Management - Console
+
+* Brand new menu to unify the experience between all Gravitee products
+* Introducing v1 APIs reaching end of support and help users migrate to v2
+* APIs list page navigation enhancements
+* APIs are read-only if managed by our GKO (Gravitee Kubernetes Operator)
+* Allow to delete a media using the Management API
+* [BETA v4 M-API] Manage v4 APIs subscriptions as an API Publisher & an API Consumer
```
Comment (ivan-gravitee) [inline pages/apim/3.x/changelog/changelog-3.20.adoc:25]:
```suggestion
* [V4 BETA M-API] Manage v4 APIs subscriptions as an API Publisher & an API Consumer
```

File: pages/apim/3.x/changelog/changelog-3.20.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+For upgrade instructions, please refer to https://docs.gravitee.io/apim/3.x/apim_installguide_migration.html[APIM Migration Guide]
+
+*Important:* If you plan to skip versions when you upgrade, ensure that you read the version-specific upgrade notes for each intermediate version. You may be required to perform manual actions as part of the upgrade.
+
+// NOTE: Global 3.20 release info here
+
+// <DO NOT REMOVE THIS COMMENT - ANCHOR FOR FUTURE RELEASES>
+
+== APIM - 3.20 (2022-12-30)
```
Comment (AndrewMcFarlandCampbellGravitee) [inline pages/apim/3.x/changelog/changelog-3.20.adoc:16]:
```suggestion
== APIM - 3.20 (2023-01-05)
```

**Comment by AndrewMcFarlandCampbellGravitee** [comment]
@mouligno @gaetanmaisse - do you have any changes?

**Comment by gaetanmaisse** [comment]
Looks good for me

**Review by mouligno — APPROVED** [review]

**Comment by ivan-gravitee** [comment]
@AndrewMcFarlandCampbellGravitee Can you please add links to the event-native docs and the GKO docs in the respective sub-sections?

**Comment by AndrewMcFarlandCampbellGravitee** [comment]
> @AndrewMcFarlandCampbellGravitee Can you please add links to the event-native docs and the GKO docs in the respective sub-sections?

Done.

**Review by ivan-gravitee — APPROVED** [review]
LGTM

**Pull request merged**